content/o-nas.md

```diff
@@ -68,6 +68,39 @@ Chceme, aby mal každý študent šancu dostať sa k našim aktivitám bez ohľa
     {{</ landing/icon-grid >}}
 {{</ container >}}
 
+{{< container class="mt-6" >}}
+    {{< landing/icon-grid >}}
+        {{< landing/icon-grid-item icon="mdi:file-document-multiple"
+            title="Výročné správy" >}}
+        - __<u>[Výročná správa za rok 2019 (pdf)](/images/vyrocne-spravy/trojsten_vs_19.jpg)</u>__
+        - __<u>[Výročná správa za rok 2018 (pdf)](/images/vyrocne-spravy/trojsten_vs_18.jpg)</u>__
+        - __<u>[Výročná správa za rok 2017 (pdf)](/images/vyrocne-spravy/trojsten_vs_17.jpg)</u>__
+        - __<u>[Výročná správa za rok 2016 (pdf)](/images/vyrocne-spravy/trojsten_vs_16.jpg)</u>__
+        - __<u>[Výročná správa za rok 2015 (pdf)](/images/vyrocne-spravy/trojsten_vs_15.jpg)</u>__
+        - __<u>[Výročná správa za rok 2014 (pdf)](/images/vyrocne-spravy/trojsten_vs_14.jpg)</u>__
+        - __<u>[Výročná správa za rok 2013 (pdf)](/images/vyrocne-spravy/trojsten_vs_13.jpg)</u>__
+        {{</ landing/icon-grid-item >}}
```

Comment:
asi by som spravil zvlast podstranku s tym zoznamom, lebo sa to bude len rozsirovat

Reply:
a bude? akože, že za tých ďalších ~10 rokov, čo tam toho bude priveľa budeme mať aj tak novú stránku..

Reply:
alebo len roky za sebou? ze <u>2021</u>, <u>2022</u>... ? mozno v opacnom poradi, ale chapess